roles/set_openstack_containers/templates/update_env_vars.sh.j2

```diff
@@ -24,7 +27,11 @@ export PATH="{{ cifmw_path }}"
 {#   It filters out the same and update the container address with new url #}
 {%   elif (_container_prefix in _container) and ('operator' not in _container) -%}
 {%     set _container_data = _container.split(_container_prefix)[-1] -%}
-{%     set _container_name = _container_data.split('@sha256')[0] -%}
+{%     if cifmw_set_openstack_containers_openstack_version_change | bool -%}
+{%        set _container_name = _container_data.split(':')[0]            -%}
```

Comment:
Maybe a little comment explaining why the parsing has to change.

Reply:
done, it has been improved and explanation added